services/web/server/tests/unit/isolated/test_projects__db_utils.py

```diff
@@ -0,0 +1,438 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+import datetime
+import json
+import re
+from copy import deepcopy
+from dataclasses import dataclass
+from itertools import combinations
+from typing import Any, Callable
+
+import pytest
+from faker import Faker
+from models_library.projects_nodes import Node
+from models_library.services import ServiceKey
+from models_library.users import GroupID, UserID
+from models_library.utils.fastapi_encoders import jsonable_encoder
+from simcore_postgres_database.models.groups import GroupType
+from simcore_service_webserver.projects._db_utils import (
+    DB_EXCLUSIVE_COLUMNS,
+    SCHEMA_NON_NULL_KEYS,
+)
+from simcore_service_webserver.projects.db import (
+    ANY_USER,
+    ProjectAccessRights,
+    assemble_array_groups,
+    check_project_permissions,
+    convert_to_db_names,
+    convert_to_schema_names,
+    create_project_access_rights,
+    patch_workbench,
+    update_workbench,
+)
+from simcore_service_webserver.projects.exceptions import (
+    NodeNotFoundError,
+    ProjectInvalidRightsError,
+    ProjectInvalidUsageError,
+)
+
+
+@pytest.fixture
+def fake_schema_dict():
+    return {
+        "anEntryThatUsesCamelCase": "I'm the entry",
+        "anotherEntryThatUsesCamelCase": "I'm also an entry",
+    }
+
+
+@pytest.fixture
+def fake_db_dict():
+    return {
+        "an_entry_that_uses_snake_case": "I'm the entry",
+        "another_entry_that_uses_snake_case": "I'm also an entry",
+    }
+
+
+def test_convert_to_db_names_transform_casing(fake_schema_dict):
+    db_entries = convert_to_db_names(fake_schema_dict)
```

Comment:
thx for these tests. They will help in the transition of this part to pydantic. I am surprised we did not have them tested!

Reply:
we did I moved them in the isolated section